.devcontainer/scripts/setup-local-settings.sh

```diff
@@ -0,0 +1,9 @@
+#! /bin/bash
+set -e
+
+
+cp ./source/workflow-designtime/starter.settings.json ./source/workflow-designtime/local.settings.json
```

Comment:
Since this is in the `postCreateCommand`, this will run each time the dev container is built. Would it make sense to only copy if the file doesn't exist? I.e. don't overwrite if already there as the user has potentially added to the settings?

Reply:
Or.... could it just live as `local.settings.json` as a starting point?